apps/portal/src/content/docs/components/drawer.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: Drawer
+description: Drawer component with stacking
+---
+
+The Drawer component works similar to a Sheet component, but allows animated stacking as well.
+
+import { DocsPage } from "@/components/docs-page";
+
+<DocsPage.ComponentExample
+  client:only
+  code={`<Drawer.Root direction='right'>
+    <Drawer.Trigger><Button>Open Drawer</Button></Drawer.Trigger>
```

Comment:
This needs to have `asChild` for the button itself to trigger things, otherwise the click event will bubble up the tigger to act on.
```suggestion
    <Drawer.Trigger asChild><Button>Open Drawer</Button></Drawer.Trigger>
```